data/headerNavigation.ts

```diff
@@ -1,6 +1,9 @@
+import { CountryVariant } from "utils/locales";
+
 interface HeaderNavigation {
   name: string;
   link: string;
+  onlyForLanguageVariants?: CountryVariant
```

Comment:
So this is okay for now but what happens when we have 2 language variants with WIkis? This needs to be an array `CountryVariant[]`. 

Then you check if the current variant is in this array `onlyForLanguageVariants.includes(getCountryVariant())`

Reply:
fixed